File: programacao/python/respostas/operacoes/39.py
bandeirada = float(input("Valor da bandeirada: "))
valor_por_250m = float(input("Valor a cada 250m: "))
distancia_km = float(input("Distancia percorrida (km): "))

total = bandeirada + (distancia_km * 1000 / 250) * valor_por_250m
print(f"O valor total da corrida e R$ {total:.2f}")

File: programacao/c/respostas/operacoes/39.c
#include <stdio.h>

int main() {
    float bandeirada, valor_por_250m, distancia_km, total;

    printf("Digite o valor da bandeirada: ");
    scanf("%f", &bandeirada);

    printf("Digite o valor a cada 250m: ");
    scanf("%f", &valor_por_250m);

    printf("Digite a distancia percorrida (km): ");
    scanf("%f", &distancia_km);

    total = bandeirada + (distancia_km * 1000 / 250) * valor_por_250m;

    printf("O valor total da corrida e R$ %.2f\n", total);

    return 0;
}
